fixture_magic/utils.py

```diff
@@ -54,15 +59,14 @@ def get_m2m(obj):
 
 def serialize_fully():
     index = 0
-
+    handlers = prepare_handlers()
+    default_handler = BaseModelHandler()
     while index < len(serialize_me):
-        for field in get_fields(serialize_me[index]):
-            if isinstance(field, models.ForeignKey):
-                add_to_serialize_list(
-                    [serialize_me[index].__getattribute__(field.name)])
-        for field in get_m2m(serialize_me[index]):
-            add_to_serialize_list(
-                serialize_me[index].__getattribute__(field.name).all())
+        full_model_name = get_instance_model_full_name(serialize_me[index])
+        if full_model_name in handlers:
+            handlers[full_model_name].handle(serialize_me[index])
+        else:
+            default_handler.handle(serialize_me[index])
```

Comment:
or to avoid if else statement:

```suggestion
        handler = handlers.get(full_model_name, default_handler)
        handler.handle(serialize_me[index])
```